### ISSUE TITLE
Firefly III svg broken?

### ISSUE THREAD

**Issue by Skaronator** [walkxcode/dashboard-icons#421]
### Icon Name

https://github.com/walkxcode/dashboard-icons/blob/main/svg/firefly.svg

The icon seems to be broken. It's there when you open the file fullscreen but seems like its out of the "box".

When using the Icon in Homepage I get an empty Icon

![image](https://github.com/walkxcode/dashboard-icons/assets/2886913/f5f12344-fa9b-45b1-88f0-4858b115ecd2)

**Comment by walkxcode** [comment]
I will take a look at this in a couple of days!

**Comment by Skaronator** [comment]
zigbee2mqtt.svg is also broken. It looks like the CI commit broke it because a previous version looks fine:

Previous: https://github.com/walkxcode/dashboard-icons/blob/9d9e48b7c05ad5a57d2d91e037dac07d062de244/svg/zigbee2mqtt.svg
Current: https://github.com/walkxcode/dashboard-icons/blob/main/svg/zigbee2mqtt.svg

**Comment by walkxcode** [comment]
Yup.. I think you are right. Do you know of an easy way to revert all SVGs (except completely new ones) back to their versions before the CI changes?

**Comment by Skaronator** [comment]
If it's just that single commit that causes all the changes then I think git reset for the SVG folder one commit before that would be enough:

https://stackoverflow.com/a/6119127

Cannot test or check since I'm on my phone.

**Comment by Skaronator** [comment]
Just asked chatGPT and yeah git revert instead of reset would be easier. (git reset would also work but more commands)

To revert a commit for a specific folder in Git, follow these steps:

1. Identify the commit hash that you want to revert. You can do this by running `git log` and copying the hash code of the commit that made changes to the folder.

2. Run the `git revert` command and provide the commit hash. For example, to revert commit "abcd123" for a folder named "my-folder", run:  

```
git revert abcd123 -- my-folder
```

3. Git will open a new file where you can enter your commit message. You can leave this file as it is and save it.

4. Push the changes to your Git remote repository using the `git push` command.

This will undo the changes made by the specified commit in the specified folder.

**Comment by walkxcode** [comment]
Wouldn't this also remove all the new SVGs that I have added manually?

**Comment by Skaronator** [comment]
Mh yeah. That's why separating different stuff into separate commits is important 😅

**Comment by walkxcode** [comment]
It is different commits, I am just wondering if this will reset all the way back to that commit, or will it just revert that single commit.

**Comment by Skaronator** [comment]
It will just revert that single commit. You might have merge conflicts but that should be easily resolvable

**Comment by walkxcode** [comment]
I am unable to just revert the SVG folder, and I'm having loads of trouble resolving merge conflicts. Can someone help me with this?
